components/dropdown/Dropdown.js

```diff
@@ -9,10 +9,13 @@ import { noop } from 'proton-shared/lib/helpers/function';
 const Dropdown = ({
     anchorRef,
     children,
+    className,
     originalPlacement = 'bottom',
     onClose = noop,
     isOpen = false,
     narrow = false,
+    wide = false,
```

Comment:
not a fan of conflicting props - it's not immediately clear what the result of `wide=true` & `narrow=true` is

`size: 'narrow' | 'wide'` is probably better